.wuf/generator/templates/Component/Component.e2e.ts

```diff
@@ -20,8 +17,7 @@ describe('{%componentName%}', () => {
   beforeEach(() => browser.get(storyUrl));
 
   it('should render', async () => {
-    const driver = {%componentName%}TestkitFactory({ dataHook });
-    await waitForVisibilityOf(await driver.element(), 'Cannot find {%ComponentName%}');
-    expect((await driver.element()).isDisplayed()).toBe(true);
+    // If the component needs a real browser environment for testing, use this file
+    // If not, please remove this file
```

Comment:
Maybe you should keep the code as well. I think it's a good reference.

Reply:
I was concerned that if there'll be some code here, it will discourage developers to delete the file 😄